bin/pg_repack.c

```diff
@@ -600,13 +628,27 @@ is_requested_relation_exists(char *errbuf, size_t errsize){
 		if (iparam < num_relations)
 			appendStringInfoChar(&sql, ',');
 	}
+	for (cell = exclude_table_list.head; cell; cell = cell->next)
+	{
+		appendStringInfo(&sql, "($%d, 'r')", iparam + 1);
+		params[iparam++] = cell->val;
+		if (iparam < num_relations)
+			appendStringInfoChar(&sql, ',');
+	}
 	for (cell = parent_table_list.head; cell; cell = cell->next)
 	{
 		appendStringInfo(&sql, "($%d, 'p')", iparam + 1);
 		params[iparam++] = cell->val;
 		if (iparam < num_relations)
 			appendStringInfoChar(&sql, ',');
 	}
+	for (cell = exclude_parent_table_list.head; cell; cell = cell->next)
```

Comment:
Same as above `exclude-parent-table` conflicts with `table` and `parent-table`:
https://github.com/reorg/pg_repack/pull/360/files#diff-a9d4d5b8968d339c50f089a6b531b1f36eedd27cdc008bdb26b70fb9fd0eee2fR382

Reply:
Same as above.

This logic helps pg_repack to find the exclude-parent-table exists in the database or not.

For example consider below case.

`pg_repack -p 5433 -d postgres  -Y public.ptable2`

In above command user can skip a parent table from a database